### PR TITLE
add footer to home page and switch current to returning

### PR DESCRIPTION
**Pull request by kweav**
Per Issue #13, switching the language from current user to returning user. This required changes to the codebook (e.g., column names), tidying data steps, the main and extra analyses, and the descriptions throughout. 

Tested the tidy, main, and extra Rmd's locally to verify that they knit properly and the plots looked as expected.

Also edited the flowchart (housed on a google slide), redownloaded the image, added it here, and added a commented out link to the slide deck on the homepage.

Added a footer to only the homepage about funding. This required edits to the style.css file as well as adding a footer.html file, and editing the YAML metadata header of the homepage.

### CODE REVIEW

**Comment by kweav** [comment]
The preview won't show the flowchart, but that's because it's pointing to the link for the image when it is on main. That's also the URL error. But I've double checked the URL will be correct.

Not sure why I don't see the footer on the preview

File: resources/dictionary.txt [hunk 1]
```diff
@@ -16,16 +16,16 @@ codebook
 CSER
 CSHL
 css
-CurrentAnVIL
```
Comment (avahoffman) [inline resources/dictionary.txt:19]: An option to keep these variable names from triggering the spell check is to use backticks, eg, ``` `CurrentAnVIL` ```.. this has the added benefit of formatting it like code, as well.

File: pages/index.Rmd [hunk 2]
```diff
@@ -1,15 +1,16 @@
 ---
 title: "**About the AnVIL Community Poll**"
 output:
-  html_document
+  html_document:
+    after_body: resources/footer.html
```
Comment (avahoffman) [inline pages/index.Rmd]: Let's move this to the _site.yml file

File: pages/resources/footer.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<footer>
+  <p>The AnVIL is supported through cooperative agreement awards from NHGRI with co-funding from OD/ODSS to the Broad Institute (U24HG010262) and Johns Hopkins University (U24HG010263). This work is part of the <a href="https://anvilproject.org/team/working-groups#outreach-working-group">AnVIL Outreach Working Group's</a> User Research.<br><br>
```
Comment (avahoffman) [inline pages/resources/footer.html:4]:
```suggestion
  <br>
  <hr>
  <p>The AnVIL is supported through cooperative agreement awards from NHGRI with co-funding from OD/ODSS to the Broad Institute (U24HG010262) and Johns Hopkins University (U24HG010263). This work is part of the <a href="https://anvilproject.org/team/working-groups#outreach-working-group">AnVIL Outreach Working Group's</a> User Research.<br><br>
```

Comment (avahoffman) [reply pages/resources/footer.html:4]: Adding a horizontal separator

**Review by avahoffman — APPROVED** [review]
Looks good, just a few minor changes! I think adding the footer to all pages (via _site.yml) should resolve the preview issue. 

Missed occurrences in `README.md`:
```
  * `IdentifyTypeOfUsers.Rmd` -- analysis for the first question of the poll that allowed us to identify respondents as either potential or current users of the AnVIL.
```
```
analysis for poll questions asked only to current AnVIL users
```

**Comment by kweav** [comment]
> Looks good, just a few minor changes! I think adding the footer to all pages (via _site.yml) should resolve the preview issue.
> 
> Missed occurrences in `README.md`:
> 
> ```
>   * `IdentifyTypeOfUsers.Rmd` -- analysis for the first question of the poll that allowed us to identify respondents as either potential or current users of the AnVIL.
> ```
> 
> ```
> analysis for poll questions asked only to current AnVIL users
> ```

Thanks for finding these and pointing them out!

**Comment by kweav** [comment]
@avahoffman do you think the footer has small enough text in the preview or should it be smaller?

**Comment by avahoffman** [comment]
> @avahoffman do you think the footer has small enough text in the preview or should it be smaller?

I might make it a size smaller than the main text. Thanks!

File: styles.css [hunk 4]
```diff
@@ -36,4 +36,10 @@
   background-color: RoyalBlue;
 }
 
+/* Footer */
 
+.footer {
+  font-family: "Lora", serif;
+  font-size: .85em;
```
Comment (kweav) [inline styles.css]:
```suggestion
  font-size: .65em;
```

**Comment by kweav** [comment]
Looking at the preview after the change I made to reduce the footer size --  I'm not sure it actually reduced the text size. Do you know what I might be missing?

**Pull request merged**